packages/maskbook/src/plugins/ITO/UI/ITO.tsx

```diff
@@ -305,10 +304,9 @@ export function ITO(props: ITO_Props) {
     //#endregion
 
     const retryITOCard = useCallback(() => {
-        retryPoolPayload()
         retryPoolTradeInfo()
         retryAvailability()
-    }, [retryPoolPayload, retryPoolTradeInfo, retryAvailability])
```

Comment:
We need a `retryPoolPayload()` when there's no error.

Reply:
You can add a separate hooks that removes the cache and trigger an update. I'll make this pattern mote general later

Reply:
I'd like to add a `forceUpdate()`

https://stackoverflow.com/a/55862077/4803321

Reply:
Is it best practice to use forceUpdate here? I don't know enough about Suspense, but forceUpdate gives me a gut feeling that it will cause the whole component tree to re-render🤔

Reply:
Please ignore it, I've thought about it for a while and there seems to be no better way to trigger update here

Reply:
@nuanyang233 yes it will let the whole tree re-render and trigger new suspense. If we wrap it in `startTransition`, it will make the process smoother.